hack/generate-proto.sh

```diff
@@ -1,121 +1,28 @@
 
-# Either protoc-gen-go, protoc-gen-gofast, or protoc-gen-gogofast can be used to build
-# server/*/<service>.pb.go from .proto files. golang/protobuf and gogo/protobuf can be used
-# interchangeably. The difference in the options are:
-# 1. protoc-gen-go - official golang/protobuf
-go build -i -o dist/protoc-gen-go ./vendor/github.com/golang/protobuf/protoc-gen-go
-GOPROTOBINARY=go
-# 2. protoc-gen-gofast - fork of golang golang/protobuf. Faster code generation
-go build -i -o dist/protoc-gen-gofast ./vendor/github.com/gogo/protobuf/protoc-gen-gofast
-GOPROTOBINARY=gofast
-# 3. protoc-gen-gogofast - faster code generation and gogo extensions and flexibility in controlling
-# the generated go code (e.g. customizing field names, nullable fields)
-go build -i -o dist/protoc-gen-gogofast ./vendor/github.com/gogo/protobuf/protoc-gen-gogofast
-GOPROTOBINARY=gogofast
-#
-## protoc-gen-grpc-gateway is used to build <service>.pb.gw.go files from from .proto files
-go build -i -o dist/protoc-gen-grpc-gateway ./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-## protoc-gen-swagger is used to build swagger.json
-go build -i -o dist/protoc-gen-swagger ./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-#
-## Generate server/<service>/(<service>.pb.go|<service>.pb.gw.go)
-PROTO_FILES=$(find $PROJECT_ROOT \( -path '*/pkg/*' -and -name "*.proto" \))
-for i in ${PROTO_FILES}; do
-    # Path to the google API gateway annotations.proto will be different depending if we are
-    # building natively (e.g. from workspace) vs. part of a docker build.
-    if [ -f /.dockerenv ]; then
-        GOOGLE_PROTO_API_PATH=$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis
-        GOGO_PROTOBUF_PATH=$GOPATH/src/github.com/gogo/protobuf
-    else
-        GOOGLE_PROTO_API_PATH=${PROJECT_ROOT}/vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis
-        GOGO_PROTOBUF_PATH=${PROJECT_ROOT}/vendor/github.com/gogo/protobuf
-    fi
+for f in $(find pkg -name '*.proto'); do
     protoc \
-        -I${PROJECT_ROOT} \
-        -I/usr/local/include \
-        -I./vendor \
-        -I$GOPATH/src \
-        -I${GOOGLE_PROTO_API_PATH} \
-        -I${GOGO_PROTOBUF_PATH} \
-        --${GOPROTOBINARY}_out=plugins=grpc:$GOPATH/src \
-        --grpc-gateway_out=logtostderr=true:$GOPATH/src \
+        -I /usr/local/include \
+        -I . \
+        -I ${GOPATH}/src \
+        -I ${GOPATH}/pkg/mod/github.com/gogo/protobuf@v1.3.1/gogoproto \
+        -I ${GOPATH}/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.12.1/third_party/googleapis \
+        --gogofast_out=plugins=grpc:${GOPATH}/src \
+        --grpc-gateway_out=logtostderr=true:${GOPATH}/src \
         --swagger_out=logtostderr=true:. \
-        $i
+        $f
 done
-
-# collect_swagger gathers swagger files into a subdirectory
```

Comment:
Argo don't need swagger scheme any more ?

Reply:
we need anther PR for this